### PR TITLE
docs: add See Also for description of export/import and save/load

### PR DESCRIPTION
**Pull request by chenhanxiao**
make them reference to each other.

Signed-off-by: Chen Hanxiao chenhanxiao@cn.fujitsu.com

### CODE REVIEW

**Comment by chenhanxiao** [comment]
@thaJeztah
As your comments in #11091, here it is.

**Comment by thaJeztah** [comment]
Great! Erm, could you change the title of this PR, because it is named after login/logout and start/stop :smile:

You should be able to change the title here on GitHub, no need to update the PR itself.

**Comment by chenhanxiao** [comment]
@thaJeztah
Thanks for your remind, I used a wrong commit message file.
I updated PR as well for the title of my PR is also wrong : (

**Comment by thaJeztah** [comment]
Oh, missed that, good catch, thanks!

File: docs/man/docker-load.1.md [hunk 1]
```diff
@@ -36,6 +36,9 @@ Restores both images and tags.
     fedora              heisenbug           58394af37342        7 weeks ago         385.5 MB
     fedora              latest              58394af37342        7 weeks ago         385.5 MB
 
+# See also
+**docker-save(1)** to save an image(s) to a tar archive.
```
Comment (thaJeztah) [inline docs/man/docker-load.1.md]: Should this mention STDOUT as well, as in the [CLI docs](https://docs.docker.com/reference/commandline/cli/#save)?

**Comment by thaJeztah** [comment]
One minor nit (it's only a "see also", not the actual help, so not sure it needs to be added)

Other than that LGTM (not a maintainer)

File: docs/man/docker-load.1.md (hunk 1 above)
Comment (chenhanxiao) [inline docs/man/docker-load.1.md]: @thaJeztah

I ommited things inside parentheses.
Let's keep it as docker-save man page said.

Thanks.

**Comment by SvenDowideit** [comment]
LGTM - @fredlf @jamtur01 @spf13 @moxiegirl

File: docs/man/docker-load.1.md [hunk 2]
```diff
@@ -36,6 +36,9 @@ Restores both images and tags.
     fedora              heisenbug           58394af37342        7 weeks ago         385.5 MB
     fedora              latest              58394af37342        7 weeks ago         385.5 MB
 
+# See also
+**docker-save(1)** to save an image(s) to a tar archive(streamed to STDOUT by default).
```
Comment (fredlf) [inline docs/man/docker-load.1.md]: Missing space after "archive"

**Comment by jamtur01** [comment]
LGTM barring @fredlf's niggle.

**Comment by fredlf** [comment]
Yes, also LGTM but for niggle.

**Comment by chenhanxiao** [comment]
@fredlf @jamtur01

Updated as your comments, please check.

**Comment by jamtur01** [comment]
LGTM

**Comment by fredlf** [comment]
LGTM

**Pull request merged**